### PR TITLE
Create contributing.md issue no(#26)

### PR DESCRIPTION
**Pull request by rajatsinghal02**
i have created the contributing.md file.

### CODE REVIEW

**Comment by rajatsinghal02** [comment]
@soham0005 please check the contributing.md file i have added to the repo

**Comment by PayalKumari10** [comment]
Hello @rajatsinghal02 I wanted to share a few suggestions to help improve our CONTRIBUTING.md file and enhance the contribution process. Here are some points to consider:

1.  **Duplicate Content:**

It would be great to remove the repeated section starting from <h1 align="center">Contributors Guide for ElectroKart⚡</h1> to keep the document concise.

2.  **Branch Naming:**

For clarity, I suggest using branch names like feature/<feature_name> or fix/<issue_name>. This will help everyone understand the purpose of the branches at a glance.

3. **Commit Messages:**

It might be helpful to encourage the use of descriptive commit messages. For example:    **_git commit -m "feat: added new feature"_**

4.**Prerequisites Section:**

Including a section that lists the required tools (like Node.js and npm versions) could be beneficial for new contributors.

5.**Testing:**

A friendly reminder to run tests before pushing changes would ensure our code stays robust:  **_npm run test_**

6. **Issue Tracker:**

It might be useful to ask contributors to assign an issue to themselves before they start working on it. This helps keep track of progress.


7. **PR Descriptions:**

Suggesting that contributors include detailed PR descriptions, especially with screenshots for UI changes, could enhance our review process.

8. **Fix Commit Block:**

Lastly, we should consider removing the **css** block around the commit step and instead use a **bash** block for better clarity.



Thank you all for your hard work and contributions! If you have any questions or thoughts about these suggestions, feel free to reach out. 😊

**Comment by rajatsinghal02** [comment]
@PayalKumari10 I have updated all these in a while.

**Comment by PayalKumari10** [comment]
> @PayalKumari10 I have updated all these in a while.

Hey @rajatsinghal02 ,

Could you please review the points I shared earlier? Some of them are important to add, while others need correcting in the file. Once these updates are made, I’ll go ahead and merge them into the repository.

Thanks so much for your effort!

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+```bash
+npm start
+```
+8. Stage your changes and commit.
+```bash
+git add .
+```
+```css
+
+git commit -m "Added new feature/fixed bug."
```
Comment (PayalKumari10) [inline contributing.md:38]: this line

**Review by PayalKumari10 — CHANGES_REQUESTED** [review]

File: contributing.md [hunk 2]
```diff
@@ -1,65 +1,78 @@
...
-1. Please fill the ***PR Template*** properly while making a Pull Request.
-2. Never commit in the `main` branch.
-3. Your work must be original, written by you not copied from other resources.
-4. You must comment on your code where necessary.
+1. Please fill out the PR template properly when creating a pull request.
+2. Assign yourself to the issue you’re working on to help us track progress.
+3. Never commit to the main branch.
+4. Your work must be original and adhere to the project's coding standards.
+5. Comment your code where necessary for clarity.
+6. Always ensure all tests pass before pushing your changes.
```
Comment (PayalKumari10) [inline contributing.md:68]: great job  @rajatsinghal02 , on the updates! Please add “Always ensure all tests pass before pushing your changes by running: npm run test” to enhance clarity for new contributors. Thank you!

**Review by PayalKumari10 — CHANGES_REQUESTED** [review]

**Comment by PayalKumari10** [comment]
> @PayalKumari10 File Updated

Hello @rajatsinghal02 , Thank you for solving and updating the issues! Great job! Happy coding, and we look forward to you tackling more challenges with us!

**Comment by PayalKumari10** [comment]
Hi @soham0005 , I have reviewed the changes, and everything looks good.  Could you please merge this PR? 
Thank you! 
Best,
@PayalKumari10

**Comment by soham0005** [comment]
@rajatsinghal02 please do make the necessary changes to pass all the test cases!

**Review by PRathod27 — APPROVED** [review]

**Review by soham0005 — APPROVED** [review]

**Pull request merged**